setup.py

```diff
@@ -58,6 +58,7 @@ def readme():
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.4',
```

Comment:
FYI - 

python 3.4 EOL'ed earlier this year - https://www.python.org/downloads/release/python-3410/ 

python 3.3 EOL'd in 2017 - https://www.python.org/downloads/release/python-337/

Reply:
This makes sense, but we're pretty conservative with removing support for old Python versions because some of the linux vendors keep supporting them well past the upstream EOL.